blank-canvas-blocks/functions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+if ( ! function_exists( 'blank_canvas_blocks_support' ) ) :
+	function blank_canvas_blocks_support()  {
+
+		// Alignwide and alignfull classes in the block editor.
+		add_theme_support( 'align-wide' );
+
+		// Add support for experimental link color control.
+		add_theme_support( 'experimental-link-color' );
+
+		// Add support for editor styles.
+		add_theme_support( 'editor-styles' );
+
```

Comment:
If editor styles are supported, do we also need to `add_theme_support( 'wp-block-styles' )` to avoid mismatches between the front-end and view?

```suggestion
		// Add support for editor styles.
		add_theme_support( 'wp-block-styles' );
```